### PR TITLE
Fix a woopsie in snapshot docs

### PR DESCRIPTION
**Pull request by ChillerDragon**
Typodragon back in the game

### CODE REVIEW

**Comment by murpii** [comment]
might aswell fix applyed > applied ^^

File: src/engine/docs/snapshots.txt [hunk 1]
```diff
@@ -33,7 +33,7 @@ Section: In depth
 
 Topic: Compression
 
-After a snapshot have been created, compression is applyed to reduce the bandwidth. There are several steps taken inorder to reduce the size of the size of the snapshot.
+After a snapshot have been created, compression is applyed to reduce the bandwidth. There are several steps taken inorder to reduce the size of the snapshot.
```
Comment (def-) [inline src/engine/docs/snapshots.txt:36]: inorder -> in order

Comment (def-) [reply src/engine/docs/snapshots.txt:36]: have been -> has been

**Comment by ChillerDragon** [comment]
Wowo @def- and @murpii english pros! :)

By the way thats a nice use case for the github suggestions feature. If you hover over a line in the changes you can click the blue plus and inset a suggestion directly from the github UI. Then the pullrequest author can merge it in with one click.

image

**Pull request merged**